Formula/libffi.rb

```diff
@@ -35,6 +35,15 @@ def install
     system "./configure", "--disable-debug", "--disable-dependency-tracking",
                           "--prefix=#{prefix}", *extra_args
     system "make", "install"
+
+    # Move lib64/* to lib/ on Linux
+    on_linux do
```

Comment:
We should aim to have the minimal modifications possible to install. If `make install` doesn't do what is required, it should be reported upstream instead of fixed in the formula.

Reply:
I'm just a middleman; @iMichka please advise

Reply:
Some linux builds will try to install stuff to `lib64` (some people thought it would be cool to have multiple lib folders on the same system, and it brought endless pain in the linux build world).

On linux we have only `lib`, like brew on Mac.

Sometimes you can add --libdir=#{lib} to the configure flags instead. Can you test this?

Reply:
It seems moving stuff afterwards is not a great solution. Or if it's a generic issue, systematically do it in `brew`?

Reply:
Libraries are installed in `#{lib}` with and without `--libdir=#{lib}` on WSL Ubuntu 20.04

```console
% uname -a
Linux windows-laptop 4.19.128-microsoft-standard #1 SMP Tue Jun 23 12:58:10 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

Reply:
Ubuntu is fine. This hits mostly fedora/centos systems. I would add the `--libdir=#{lib}` flag to the Mac build too. It does not harm on Mac and on Ubuntu (basically it changes nothing), and it helps for the poor few running linux on these lib64/lib32 systems. I think this is the nicest solution and does not introduce too much code.

If we decide not to that, then we need to drop support for these systems in our documentation.

Reply:
Does that mean we end up with `--libdir=#{lib}` in almost every autoconf-based formula? That's an awful lot.

I would really suggest fixing it at the meta level. Either symlink lib64 to lib, or make brew aware of lib64?

Reply:
>That's an awful lot.

614, to be precise.

>make brew aware of lib64

I like this idea, but I don't know how much work it'll be.

Reply:
Can we add a new `std_configure_args` DSL. I bet we are setting --prefix 614 times too.
We have set the --libdir flag in the `std_meso_args`.

Adding the DSL is straightforward in brew, just copy/modify the meson one.

Could someone start that work? I won't have any time as this is quite a long project. It's easy, but it's just long.